### DISCUSSION TITLE
license status /[isp] dhcp-status /wan-lan-ip-conflict

### DISCUSSION THREAD

**Discussion by magarjoba** [FortiPower/PowerFGT#257] [Ideas]
hi
colleagues, faced with the need due to a large fleet of devices. to collect information on licensing...
yes, there is a portal) but when there is a usability filtered dashboard (much more convenient)

in general, I found the api api/v2/monitor/license/status
manually tested, suitable for versions 6.0.0-7.2.7 (40F 30E 30D 60F) tested devices..

I propose to add 3 functions.
$server = "test123"
$fgthead = Connect-FGT -Server $server -Username $Username -Password $Password -SkipCertificateCheck -DefaultConnection:$false -Timeout 15

1. view the status of the current license.
```
$responselicense = Invoke-FGTRestMethod -method "get" -uri "api/v2/monitor/license/status" -connection $fgthead
$activationstatus = $responselicense.'results'
```
-------------
2. view the gateway issued by the provider, where $ispint is your port. [ppoe / dhcp] gatawey
```
$ispint="wan"
$responsegtw = Invoke-FGTRestMethod -method "get" -uri "api/v2/monitor/system/interface/dhcp-status?mkey=$ispint" -connection $fgthead
```
-------------
3. for connoisseurs)) api/v2/monitor/system/wan-lan-ip-conflic

it happens that a great provider issues an IP address that is already occupied by your internal interface) .
if you do not have overlap registered, the interface will be rejected. therefore this status can notify about the problem in advance..
```
config system settings
set allow-subnet-overlap enable
end
```

**Comment by alagoutte** [comment]
Hi,

please push PR with this new function

**Reply by alagoutte** [reply]
@magarjoba do you plan to push a PR with this support ?

**Comment by CedricMoreau** [comment]
> 3\. for connoisseurs)) api/v2/monitor/system/wan-lan-ip-conflic

Where did you found this API ? On FNDN i can't find it https://fndn.fortinet.net/index.php?/fortiapi/1-fortios/3694/1/system/

**Comment by alagoutte** [comment]
Licence Status already supported (#164) and dhcp status add with #276

for wan-lan-ip-conflit, no found doc... (there is system/ipconf...)
